esmvalcore/cmor/_fixes/shared.py

```diff
@@ -0,0 +1,25 @@
+"""Shared functions for fixes."""
+import iris
+import numpy as np
+from cf_units import Unit
+
+
+def add_height_coord(cube):
+    """Add height coordinate to cube."""
+    height_coord = iris.coords.AuxCoord(2.0,
+                                        var_name='height',
+                                        standard_name='height',
+                                        long_name='height',
+                                        units=Unit('m'),
+                                        attributes={'positive': 'up'})
+    cube.add_aux_coord(height_coord, ())
+    return cube
+
+
+def round_coordinates(cubes, decimals=5):
+    """Round all dimensional coordinates of all cubes."""
+    for cube in cubes:
+        for coord in cube.coords(dim_coords=True):
+            for attr in ('points', 'bounds'):
+                setattr(coord, attr, np.round(getattr(coord, attr), decimals))
```

Comment:
It would be nicer to keep the data lazy:
```python
            coord.points = da.round(coord.core_points(), decimals))
            coord.bounds = da.round(coord.core_bounds(), decimals))
```

Reply:
`da.round()` raises `AttributeError: 'numpy.ndarray' object has no attribute 'map_blocks'` in its current version.

Reply:
I guess the function expects a dask array, but in your test the data is not lazy to begin with. Maybe you can try
```python
coord.points = da.round(da.asarray(coord.core_points()), decimals)
```
to get it working with both dask and numpy arrays?

Reply:
Done